superset/assets/src/logger/LogUtils.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+// Log event names ------------------------------------------------------------
+export const LOG_ACTIONS_LOAD_CHART = 'load_chart';
+export const LOG_ACTIONS_RENDER_CHART = 'render_chart';
+
+export const LOG_ACTIONS_MOUNT_DASHBOARD = 'mount_dashboard';
+export const LOG_ACTIONS_MOUNT_EXPLORER = 'mount_explorer';
+
+export const LOG_ACTIONS_SELECT_DASHBOARD_TAB = 'select_dashboard_tab';
+export const LOG_ACTIONS_RENDER_CHART_CONTAINER = 'render_chart_container';
+export const LOG_ACTIONS_REFRESH_CHART = 'force_refresh_chart';
```

Comment:
minor:
This `const` and value are not the same.

Reply:
thank you! fixed again. 😓